lib/rails_admin/config.rb

```diff
@@ -254,6 +257,7 @@ def models
       # @see RailsAdmin::Config.registry
       def reset
         @compact_show_view = true
+        @browser_validations = false
```

Comment:
I'm a bit concerned about defaulting this to `false`.
Don't you think that client-side validation is a nice-to-have feature?